### PR TITLE
add multiple capabilities by topic

### PR DESCRIPTION
**Pull request by saiaman**
Add more the one capability by topic.
This is usefull when you have a MQTT json payload with more than one data to parse

### CODE REVIEW

File: drivers/device/device.js [hunk 1]
```diff
@@ -77,7 +77,10 @@ class MQTTDevice extends Homey.Device {
             for (let capabilityId in this._capabilities) {
                 const stateTopic = this._capabilities[capabilityId].stateTopic;
                 if (stateTopic) {
-                    this._topics.set(stateTopic, capabilityId);
+                    if(!this._topics[stateTopic]){
```
Comment (harriedegroot) [inline drivers/device/device.js:80]: _topics is a Map, the get() and set() methods should be used to access its contents.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Comment (saiaman) [reply drivers/device/device.js:80]: It's not a problem. In javascript a map or an object can be accessed in this way.
I've tested it on a local node script

Comment (saiaman) [reply drivers/device/device.js]: fixed

Comment (harriedegroot) [reply drivers/device/device.js]: FYI: if you put something in a map with an indexer like store[key] = value, you effectively put it into the base object, not in the map. it wont be returned when requested with the map methods like get() or entries(). I.e. store.get(key) will return undefined.

**Pull request merged**